crates/trie/sparse/src/state.rs

```diff
@@ -75,21 +75,22 @@ impl SparseStateTrie {
         self
     }
 
+    /// Set the state root to the given `SparseTrie`.
```

Comment:
I wouldn't use the word root here, since it sets the whole trie, maybe accounts trie is more accurate
```suggestion
    /// Set the accounts trie to the given `SparseTrie`.
```